Makefile

```
@@ -55,6 +55,22 @@ helm-upgrade: cert-manager
 lint:  ## Lint helm chart using ct (chart-testing).
 	ct lint --config ct.yaml
 
+.PHONY: timescaledb-single
+timescaledb-single:
+
+.PHONY: timescaledb-single
+timescaledb-single: ## This is a phony target that is used to install the timescaledb-single chart.
+	kubectl create ns timescaledb
+	helm repo add timescaledb 'https://raw.githubusercontent.com/timescale/helm-charts/master/charts/repo/'
```

```suggestion
	helm repo add timescaledb 'https://raw.githubusercontent.com/timescale/helm-charts/main/charts/repo/'
```

The default branch now is ✨ main ✨

---

Thanks!